pkg/asset/rhcos/bootstrap_image.go

```diff
@@ -43,6 +45,13 @@ func (i *BootstrapImage) Generate(p asset.Parents) error {
 	defer cancel()
 	switch config.Platform.Name() {
 	case baremetal.Name:
+		// Check for RHCOS image URL override
+		if osimage = config.Platform.BareMetal.BootstrapMachineOSImage; osimage != "" {
+			logrus.Infof("Found an URL override in Baremetal Platform for Bootstrap Machine OS Image: %s", osimage)
```

Comment:
you don't need to print INFO for item set from the install-config API because it will be visible when we ask the user for what they used.

Reply:
The installer deletes the user install-config.yaml after it consumes it, so when something goes wrong the idea was that we'll know a user provided an override of this value when doing debug/support activities.  

Is the install-config already recorded somewhere in the log output for this purpose?

Reply:
> The installer deletes the user install-config.yaml after it consumes it

the install-config.yaml is available to user is they run `create install-config` from their existing directory.

logging known types is conflicting with all the other known configuration in the install-config... and therefore i would like to keep one way of knowing this type of information.

Reply:
Removed the log message as requested.